### ISSUE TITLE
Switch fs.promises.access for existsSync

### ISSUE THREAD

**Issue by codewithkyle** [Pageworks/papertrain#201]
https://github.com/Pageworks/papertrain/blob/fdb2ff759b57e04233366a47c9826fdb6cf35251/build-tools/prep.config.js#L28

Using the [access](https://nodejs.org/api/fs.html#fs_fspromises_access_path_mode) method is failing on production.

```
TypeError: Cannot read property 'access' of undefined
```

Switch to use [existsSync](https://nodejs.org/api/fs.html#fs_fs_existssync_path)

```javascript
if (fs.existsSync('_compiled'))
{
    rimraf('_compiled', (error)=>{
        if (error)
        {
            reject(error);
        }

        resolve();
    });
}
else
{
    resolve();
}
```

**Comment by codewithkyle** [comment]
It's possible that the error had something to do with the fact that `fs.promises` was experimental in node `v10.0.0` and it might work if we use node `v12.10.0`

**Comment by codewithkyle** [comment]
Use the code above as the working fix. During the next project launch we'll test using node v12.10.0 to see if fs.promises is working.

**Comment by codewithkyle** [comment]
Updating node to v12.10.0 didn't fix the issue, it appears that it's not failing because the `fs.promises` section of the API was experimental in v10. We should implement the solution provided in the OP.

**Comment by codewithkyle** [comment]
Released in Papertrain v0.3.1 (#211)
